ext/mysqli/mysqli_warning.c

```diff
@@ -256,9 +256,6 @@ PHP_METHOD(mysqli_warning, __construct)
 	MYSQLI_WARNING  *w;
 	MYSQLI_RESOURCE *mysqli_resource;
 
-	if (ZEND_NUM_ARGS() != 1) {
-		WRONG_PARAM_COUNT;
-	}
 	if (zend_parse_parameters(1, "o", &z)==FAILURE) {
```

Comment:
This hardcode number of parameters, shouldn't we use `ZEND_NUM_ARGS()`?

Reply:
yes

Reply:
I believe there are other cases, gonna check.